src/DynamoCoreWpf/ViewModels/Core/DynamoViewModel.cs

```diff
@@ -3498,6 +3498,90 @@ private static void LoadLibraryEvents_LoadLibraryFailure(string failureMessage,
             Wpf.Utilities.MessageBoxService.Show(failureMessage, messageBoxTitle, MessageBoxButton.OK, MessageBoxImage.Exclamation);
         }
 
+        internal string GetMinimumQualifiedName(NodeModel nodeModel)
```

Comment:
Maybe some comments to the functions? Understand that some of them are simply moved from the other class